subworkflows/local/variant_calling/call_sv_germlinecnvcaller.nf

```diff
@@ -39,28 +41,38 @@ workflow CALL_SV_GERMLINECNVCALLER {
         GATK4_GERMLINECNVCALLER ( ch_gcnvc_in )
 
         GATK4_GERMLINECNVCALLER.out.calls.toList()
-            .flatMap {reduce_input(it)}
+            .flatMap {CustomFunctions.reduce_input(it)}
```

Comment:
I'm surprise that you need to add `CustomFunctions.`, since `reduce_input()` is defined in this script. Is it just the way it is done in NF?

Reply:
Oops.. that workflow has been tested by folks in Lund, so we know it works. But this was all me. I must have missed to revert the changes as I was moving the functions around. I will fix it  😄

I do have plans to move this function to CustomsFunctions in the next release.